user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -529,3 +531,8 @@ the other [How-To Guides](/doc/#how-to-guides), and learn about
 * Please do **not** email individual members of the Qubes team with questions
   about installation or other problems. Instead, please see [Help, Support,
   Mailing Lists, and Forum](/support/) for appropriate places to ask questions.
+
+
+###### Contributor Report
+
+  1: Images are not up-to-date as R4.0 depreciated.
```

Comment:
Please don't add stuff like this to the documentation. Open an issue in qubes-issues instead.